packages/mobile/src/tokens/selectors.ts

```diff
@@ -30,21 +30,32 @@ export const tokensListSelector = createSelector(tokensByAddressSelector, (token
   return Object.values(tokens).map((token) => token!)
 })
 
-export const tokensWithBalanceSelector = createSelector(tokensListSelector, (tokens) => {
+type TokenBalanceWithUsdPrice = TokenBalance & {
+  usdPrice: BigNumber
+}
+
+export const tokensWithUsdBalanceSelector = createSelector(tokensListSelector, (tokens) => {
```

Comment:
naming nit: "USD Value" instead of "USD Balance" (i.e., `tokensWithUsdValueSelector`)

I don't really have a great reason and basing this on conversational language (we say "what's the value in USD?" vs "what's the balance in USD?")

Reply:
Hmm, I changed it but just for the record I find balance clearer than value, maybe because English is not my first language? I just don't use the word 'value' very often in this context

Reply:
I'm also on @gnardini's side here, as I'm seeing this function listing the tokens for which we have a balance that can be represented in USD.
Isn't it what this function is?

Reply:
I don't often hear "what is the balance of CELO in USD?".  Balance connotes offsetting in accounting.  E.g., "What's the number I need to balance this?"

I do often hear "what is the value of CELO in USD?".